src/encoded/static/components/dataset.js

```diff
@@ -407,6 +407,149 @@ const PublicationData = auditDecor(PublicationDataComponent);
+    const adminUser = !!(reactContext.session_properties && reactContext.session_properties.admin);
+    const experimentsUrl = `/search/?type=Experiment&possible_controls.accession=${context.accession}`;
+
+    // Build up array of documents attached to this dataset
+    const datasetDocuments = (context.documents && context.documents.length > 0) ? context.documents : [];
+
+    // Set up the breadcrumbs
+    const datasetType = context['@type'][1];
+    const filesetType = context['@type'][0];
+    const crumbs = [
+        { id: 'Datasets' },
+        { id: datasetType, uri: `/search/?type=${datasetType}`, wholeTip: `Search for ${datasetType}` },
+        { id: breakSetName(filesetType), uri: `/search/?type=${filesetType}`, wholeTip: `Search for ${filesetType}` },
+    ];
+
+    const crumbsReleased = (context.status === 'released');
+
+    // Render the publication links
+    const referenceList = pubReferenceList(context.references);
+
+    return (
+        <div className={itemClass}>
+            <header>
+                <Breadcrumbs crumbs={crumbs} crumbsReleased={crumbsReleased} />
+                <h1>Summary for computational model file set {context.accession}</h1>
+                <div className="replacement-accessions">
+                    <AlternateAccession altAcc={context.alternate_accessions} />
+                </div>
+                <ItemAccessories item={context} audit={{ auditIndicators, auditId: 'computationalmodel-audit' }} />
+            </header>
+            {auditDetail(context.audit, 'computationalmodel-audit', { session: reactContext.session, sessionProperties: reactContext.session_properties, except: context['@id'] })}
+            <Panel>
+                <PanelBody addClasses="panel__split">
+                    <div className="panel__split-element">
+                        <div className="panel__split-heading panel__split-heading--computational-model">
+                            <h4>Summary</h4>
+                        </div>
+                        <dl className="key-value">
+                            <div data-test="status">
+                                <dt>Status</dt>
+                                <dd><Status item={context} inline /></dd>
+                            </div>
+
+                            <div data-test="accession">
+                                <dt>Accession</dt>
+                                <dd>{context.accession}</dd>
+                            </div>
+
+                            {context.description ?
+                                <div data-test="description">
+                                    <dt>Description</dt>
+                                    <dd>{context.description}</dd>
+                                </div>
+                            : null}
+
+                            {context.computational_model_type ?
+                                <div data-test="type">
+                                    <dt>Annotation type</dt>
```

Comment:
Is this the right title? In the schema it’s called “Computational model type.”

Reply:
changed title